### PR TITLE
GCC 14 patchset for GTK2

### PR DESCRIPTION
**Pull request by Eonfge**

### CODE REVIEW

**Comment by Eonfge** [comment]
@bbhtt Can you also look at this one?

File: gtk2/gtk2.json [hunk 1]
```diff
@@ -67,6 +67,10 @@
         {
           "type": "patch",
           "path": "gtk2-fix-crash-in-show-uri.patch"
+        },
+        {
+          "type": "patch",
+          "path": "gtk2-gcc14.patch"
```
Comment (hfiguiere) [inline gtk2/gtk2.json:73]: nit
```suggestion
          "paths": [
              "gtk2-fix-crash-in-show-uri.patch",
              "gtk2-gcc14.patch"
          ]
```

actually merged with the previous one too (line 65) but github doesn't allow me to suggest it.

**Comment by hfiguiere** [comment]
maybe squash the patches into 1.

**Comment by bbhtt** [comment]
Tingping owns this module.

**Pull request merged**